functions/src/modules/SQR.functions.ts

```diff
@@ -101,41 +101,39 @@ export const sendEmailOnNewAllotment = functions.database.ref('/sqr/allotments/{
 //
 //      2. Remove DB entries for MP3s that don't exist (removeNonExistingMp3DBEntries)
 /////////////////////////////////////////////////
-
-exports.syncStorageToDB = functions.https.onRequest( async (req, res) => {
+export const syncStorageToDB = functions.https.onRequest( async (req, res) => {
     ///////////////////////////////////////////////////////
     //      1. Add the currently uploaded MP3s into the DB
     ///////////////////////////////////////////////////////
     const bucketFiles = await bucket.getFiles();
     bucketFiles.forEach(innerFilesObject => {
         innerFilesObject.forEach(file => {
-            helpers.storeFileNameToDB(file.name, db, 'sqr');
+            helpers.storeFileToDB(file.name, db, 'soundQualityReporting');
         });
     });
 
     ///////////////////////////////////////////////////////
     //      2. Remove DB entries for MP3s that don't exist
     ///////////////////////////////////////////////////////
-    const filesSnapshot = await db.ref(`/sqr/files`).once("value");
+    const filesSnapshot = await db.ref(`/files`).once("value");
     let files = filesSnapshot.val();
 
     for (let list in files) {
         for (let file in files[list]) {
             const existingBucketFiles = await bucket.file(`/mp3/${list}/${file}.mp3`).exists();
             // **Found** in DB but not in STORAGE
             // Removing should be done only if the `status` is `Spare`
-            if (!existingBucketFiles[0] && files[list][file].status === 'Spare') 
-                helpers.removeFromDB(db, `/sqr/files/${list}/${file}`)
+            if (!existingBucketFiles[0] && files[list][file]['soundQualityReporting'].status === 'Spare') 
+                helpers.removeFromDB(db, `/files/${list}/${file}`)
```

Comment:
I have updated the task description in Trello. You need to check both sub-objects here for the status.